### PR TITLE
Kubernetes edits + open questions

### PR DESCRIPTION
**Pull request by CathPag**
Signed-off-by: Catherine Paganini <74001907+CathPag@users.noreply.github.com>

### Describe your changes
Made some edits so it's easier to understand but still have open questions. Some sentences don't make 100% sense to me (please view comments below)

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/en/kubernetes.md [hunk 1]
```diff
@@ -7,49 +7,32 @@ tags: ["infrastructure", "", ""]
...
-allowing users to deploy applications declaratively in a reproducible way. 
-Software products and projects in the Kubernetes ecosystem take advantage of that automation and extensibility 
-to extend the Kubernetes [API](/application-programming-interface/). 
-This enables them to leverage Kubernetes’ automation and 
-make their tools more accessible to experienced Kubernetes practitioners.
+Kubernetes, often abbreviated as K8s, is an open source container orchestrator. 
+It automates the lifecycle of containerized applications on modern infrastructures, functioning as a "datacenter operating system" that manages applications across a [distributed system](/distributed-systems/).
+
+Kubernetes schedules [containers](container/) across [nodes](/nodes/) in a [cluster](/cluster/), bundling several infrastructure constructs, sometimes referred to as "primitives." 
+These primitives consist of app instances, load balancers, persistent storage, etc., and are composed into applications.
```
Comment (CathPag) [inline content/en/kubernetes.md:14]: The original text says, "composed into an application." What does that mean? Building an app?

File: content/en/kubernetes.md [hunk 2]
```diff
@@ -7,49 +7,32 @@ tags: ["infrastructure", "", ""]
...
-As demand for compute resources increases and 
-organizations feel pressured to provide more operational capabilities with fewer engineers, 
-new technologies and working methods are required to meet that demand. 
-Additionally, the rise of cloud computing was paired with [containerization](/containerization/) and 
-organizations that were busy automating more traditional infrastructure needed a mechanism to 
-automate the configuration and deployment of their containers.
+Infrastructure automation and declarative configuration management have been important concepts for a long time, but they have become more pressing as [cloud computing](cloud-computing/) has gained popularity. 
+As demand for compute resources increases and organizations need to provide more operational capabilities with fewer engineers, new technologies and working methods are required to meet that demand. 
+Additionally, the rise of cloud computing along with containerization and organizations busy automating more traditional infrastructure need a mechanism to automate the configuration and deployment of their containers.
```
Comment (CathPag) [inline content/en/kubernetes.md:24]: what does "organizations busy automating more traditional infrastructure" mean here?

File: content/en/kubernetes.md (hunk 1 above)
Comment (CathPag) [inline content/en/kubernetes.md:14]: The original text says, "composed into an application." What does that mean? Building an app?

**Review by CathPag — COMMENTED** [review]
Added my open questions

File: content/en/kubernetes.md (hunk 1 above)
Comment (seokho-son) [reply content/en/kubernetes.md:14]: @CathPag 
I think it means
The primitives (resources) are composed into an **environment** to run applications.

Comment (seokho-son) [inline content/en/kubernetes.md:14]: My suggestion to simplify sentences is as follows,

```suggestion
Kubernetes schedules [containers](container/) across [nodes](/nodes/) in a [cluster](/cluster/), bundling several infrastructure resources such as load balancers, persistent storages, and etc. to run containerized applications.
```

**Review by iamNoah1 — APPROVED** [review]
lgtm

File: content/en/kubernetes.md [hunk 3]
```diff
@@ -7,49 +7,31 @@ tags: ["infrastructure", "", ""]
...
-This enables them to leverage Kubernetes’ automation and 
-make their tools more accessible to experienced Kubernetes practitioners.
+Kubernetes, often abbreviated as K8s, is an open source container orchestrator. 
+It automates the lifecycle of containerized applications on modern infrastructures, functioning as a "datacenter operating system" that manages applications across a [distributed system](/distributed-systems/).
+
+Kubernetes schedules [containers](container/) across [nodes](/nodes/) in a [cluster](/cluster/), bundling several infrastructure resources such as load balancers, persistent storages, and etc. to run containerized applications.
+
+Kubernetes enables automation and extensibility, allowing users to deploy applications declaratively (see below) in a reproducible way. 
+Other software solutions in the Kubernetes ecosystem take advantage of that automation and extensibility.
+Kubernetes is easily extendable via its [API](/application-programming-interface/), and its automation helps make other tools more accessible to experienced Kubernetes practitioners.
```
Comment (seokho-son) [inline content/en/kubernetes.md:17]: I tried to make the sentence easier.

```suggestion
Kubernetes is extensible via its [API](/application-programming-interface/) which allows experienced Kubernetes practitioners to utilize the automation capability of Kubernetes in a customized way according to their needs.
```

File: content/en/kubernetes.md (hunk 2 above)
Comment (seokho-son) [reply content/en/kubernetes.md:23]: @CathPag 
It is ambiguous to me too. Maybe it is redundant as well considering 

> organizations need to provide more operational capabilities with fewer engineers, new technologies and working methods are required to meet that demand

mentioning a similar issue.

**Review by iamNoah1 — CHANGES_REQUESTED** [review]
spellcheck fails for `storages`

File: content/en/kubernetes.md (hunk 2 above)
Comment (CathPag) [reply content/en/kubernetes.md:22]: Ok, so we just drop the sentence entirely, right?

**Review by iamNoah1 — APPROVED** [review]
lgtm

File: content/en/kubernetes.md (hunk 2 above)
Comment (CathPag) [reply content/en/kubernetes.md]: I think that's what you meant, so I deleted it. Let me know if that works.

File: content/en/kubernetes.md [hunk 4]
```diff
@@ -7,49 +7,29 @@ tags: ["infrastructure", "", ""]
...
-that are more resistant to configuration drift than virtual or physical machines.
-Kubernetes works declaratively, which means that 
-instead of operators providing the instructions about how to do something 
-they instead describe, usually as manifest files (e.g. YAML), what they want to be done; 
-Kubernetes will take care of the "how" on its own. 
+Similar to traditional [infrastructure as code](/infrastructure-as-code/) tools, Kubernetes helps with automation but has the advantage of working with containers. 
+Containers are more resistant to configuration drift than [virtual](/virtual-machine/) or physical machines. 
+
+Additionally, Kubernetes works declaratively, which means that instead of operators instructing the machine how to do something, they describe — usually as manifest files (e.g., YAML) — what the infrastructure should look like. 
+Kubernetes then takes care of the "how."
```
Comment (seokho-son) [inline content/en/kubernetes.md:29]:
```suggestion
Kubernetes then takes care of the "how". 
```

**Review by seokho-son — APPROVED** [review]
LGTM :)

**Pull request merged**